plugins/system/kernel_same_page_merging

```diff
@@ -48,20 +45,20 @@ pages_volatile.draw AREASTACK
 EOF
 }
 
-# prüfe ob ksm aktiviert ist
-if [ ! -f "$ksm_dir_run" ]; then
-  exit 1
-fi
+
 
 # wenn parameter = ...
 if [ "$1" = "autoconf" ]; then
-  echo yes
+  # prüfe ob ksm aktiviert ist
+  if [ ! -f "$ksm_dir_run" ]; then
+    exit 1
```

Comment:
In case of an `autoconf` failure, the reason for the failure should be reported (e.g. `echo "no (missing $ksm_dir_run directory)"`) and the plugin should exit with success (see the [documentation](http://guide.munin-monitoring.org/en/latest/develop/plugins/plugin-concise.html?highlight=autoconf#autoconf)).

Reply:
changed, is that alright?